scripts/winesapos-install.sh

```diff
@@ -227,6 +229,19 @@ if [[ "${WINESAPOS_BUILD_CHROOT_ONLY}" == "false" ]]; then
     done
 
     echo "Mounting partitions complete."
+    # Mohamed: Once formatting is completed, you can now use ./pifs to convert the filesystem to πfs
+    if [[ "${WINESAPOS_PIFS}" == "true" ]]; then
+        CWD = $PWD
+        cd /tmp
+        git clone https://github.com/aya/pifs
+        cd pifs
+        ./autogen.sh
+        ./configure
+        make
+        # Mohamed: Now convert it!
+        ./pifs /
```

Comment:
I tested this an it said that a metadata directory needs to be specified. You could do something like:

```
mkdir /tmp/pifs-metadata
pifs -o mdd/tmp/pifs-metadata ./
```

This is all temporary, by the way. The mount and metadata folders are in `/tmp`. The install with `sudo make install` would also outside of the chroot.